app/api/models/audit/GenericAuditDetail.scala

```diff
@@ -37,6 +39,7 @@ object GenericAuditDetail {
       JsPath.writeNullable[Map[String, Option[String]]] and
       (JsPath \ "request").writeNullable[JsValue] and
       (JsPath \ "X-CorrelationId").write[String] and
+      (JsPath \ "versionNumber").write[String] and
```

Comment:
Unless this is specified that it has to write "versionNumber", could it also write the name as "apiVersion"?

Reply:
The ticket specified that the field name is "versionNumber" but I do think that "apiVersion" is a more suitable name. Might be worth raising this change.